bzt/modules/python.py

```diff
@@ -324,6 +388,42 @@ def gen_setup_method(self):
         elif browser == 'Chrome':
             statement = "self.driver = webdriver.Chrome(service_log_path=%s)"
             setup_method_def.append(self.gen_statement(statement % repr(self.wdlog)))
+        elif browser == 'Remote':
+
+            remote_capabilities = dict(self.scenario).get("capabilities", [])
+            remote_capabilities = remote_capabilities + inherited_capabilities
+
+            supported_capabilities = ["browser", "version", "javascript", "platform", "os_version",
+                                      "selenium", "device", "app"]
+            for capability in remote_capabilities:
```

Comment:
I'd argue that `capabilities` doesn't need to be a list of dicts and can be a single dict instead.
This would change configuration from, for example,
```yaml
scenarios:
  request_example:
    browser: Remote
    remote: http://user:key@remote_web_driver_host:port/wd/hub
    capabilities:
      - browser: firefox
      - version: "54.0"
    requests:
    - http://demo.blazemeter.com/
```
to
```yaml
scenarios:
  request_example:
    browser: Remote
    remote: http://user:key@remote_web_driver_host:port/wd/hub
    capabilities:
      browser: firefox
      version: "54.0"
    requests:
    - http://demo.blazemeter.com/
```

This would be simpler to write for user and simpler to handle in Taurus.

Reply:
Hi @dmand ! Thnks! A pleasure to collaborate with an excellent project.

I made a change that would allow us to use both methods.

I updated the documentation so that users preferably use the simplest of them (and if they use the other method by any reason, they wouldn't break anything).